production/bin/create_configs.sh

```diff
@@ -64,24 +64,6 @@ echo "Starting application configuration. Processing ENV settings."
     fi
   fi
 
-  # Populate production environment config from SSM:
```

Comment:
We no longer want to do this since it's what pulled the file out of SSM and decoded it from Base64. So, delete!